invokeai/app/api/routers/images.py

```diff
@@ -322,3 +322,17 @@ async def unstar_images_in_list(
         return ImagesUpdatedFromListResult(updated_image_names=updated_image_names)
     except Exception:
         raise HTTPException(status_code=500, detail="Failed to unstar images")
+
+
+class ImagesDownloaded(BaseModel):
```

Comment:
May be worth adding the path to where they'll be available at here. Whether a link or a filepath

Reply:
Yeah I figured whoever implements the actual working code here could return a message with what makes sense